docs/installation.md

```diff
@@ -18,13 +18,13 @@ The installation configuration should be hosted on an HTTP or TFTP server. A sim
 ### Kernel Command Line
 
 Install directives can be set from the kernel command line using a period (.) seperated key structure such as
-`Elemental.install.config_url`.  They kernel command line keys are case-insensitive.
+`elemental.install.config_url`.  They kernel command line keys are case-insensitive.
```

Comment:
Is this for cloud-config file passing via kernel cmd line 🤔 ? 
I guess this should be now:
```suggestion
`elemental.install.config-urls`.  They kernel command line keys are case-insensitive.
```
But I think @davidcassany knows better!

Reply:
Hmm, that might explain a few errors I've run into! I will try that on a new install and see if it works better!

Reply:
Seems like this i deprecated, I removed the whole section instead :+1: